app/forms/EducationQuestionnaireForm.scala

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2016 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import connectors.exchange._
+import play.api.data.Form
+import play.api.data.Forms._
+import play.api.i18n.Messages
+
+object EducationQuestionnaireForm {
+
+  val form = Form(
+    mapping(
+      "liveInUKBetween14and18" -> Mappings.nonEmptyTrimmedText("error.liveInUKBetween14and18.required", 31),
+      "postcodeQ" -> of(requiredFormatterWithMaxLengthCheckAndSeparatePreferNotToSay("liveInUKBetween14and18",
+        "postcodeQ", "preferNotSay_postcodeQ", Some(256))),
+      "preferNotSay_postcodeQ" -> optional(checked(Messages("error.required.postcodeQ"))),
+      "schoolName14to16" -> of(requiredFormatterWithMaxLengthCheckAndSeparatePreferNotToSay("liveInUKBetween14and18",
+        "schoolName14to16", "preferNotSay_schoolName14to16", Some(256))),
+      "preferNotSay_schoolName14to16" -> optional(checked(Messages("error.required.schoolName14to16"))),
+      "schoolName16to18" -> of(requiredFormatterWithMaxLengthCheckAndSeparatePreferNotToSay("liveInUKBetween14and18",
+        "schoolName16to18", "preferNotSay_schoolName14to16", Some(256))),
+      "preferNotSay_schoolName16to18" -> optional(checked(Messages("error.required.schoolName16to18"))),
+      "freeSchoolMeals" -> of(requiredFormatterWithMaxLengthCheck("liveInUKBetween14and18", "freeSchoolMeals", Some(256))),
+      "isCandidateCivilServant" -> Mappings.nonEmptyTrimmedText("error.isCandidateCivilServant.required", 31),
```

Comment:
I will add this to the technical debt document. I would like to improve all that code at some point.
https://confluence.tools.tax.service.gov.uk/display/FSET/Technical+Debt